### DISCUSSION TITLE
generated .llc file

### DISCUSSION THREAD

**Discussion by csengh** [mifi/lossless-cut#1929] [Support] [answered]
hy!
I just tried this nice tool, everything works fine but I don't know what is the generated .llc file? I see it is just a text file but what is its role?

**Comment by mifi** [comment, marked as answer]
Hi
It’s the project file, stores your cutpoints so that you dont lose them if you close the app

**Comment by csengh** [comment]
I got it. However the project file cannot be moved, it must be in the same folder as the referenced file because the full path name cannot be used in 'mediaFileName:'

thank's

**Reply by mifi** [reply]
That is correct, that is how it works
